sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonSuite.scala

```diff
@@ -481,59 +481,6 @@ abstract class JsonSuite extends QueryTest with SharedSparkSession with TestJson
     )
   }
 
-  ignore("Type conflict in primitive field values (Ignored)") {
```

Comment:
Could you give us more history about what disabled this initially?
> The test is not maintained for long time. It can be removed to reduce size of JsonSuite, and improve maintainability.

Reply:
The test was initially added as ignored one 6 years ago by https://github.com/apache/spark/commit/d2f4f30b12f99358953e2781957468e2cfe3c916#diff-054d06e340007b5abd1e0b61ccd9aa49R366

Reply:
Thanks!